report/Appendix/SemanticsTree.tex

```diff
@@ -0,0 +1,44 @@
+% !TeX root = ../main.tex
+\begin{landscape}
+\section{Semantics Derivation Tree Example}
+\label{sec:semanticsTree}
+This is an example of how to create a semantic derivation tree for a program written in HCL.
+It is based on the following code snippet.
+
+\begin{lstlisting}[language=HCL,label=lis:derivationCode,firstnumber=1]
+num x
+x = 5
+\end{lstlisting}
+
+Rewritten using the abstract syntax described in section \ref{sec:semantics}, we get the following big step semantic:
```

Comment:
the following big step semantic is produced

Reply:
still want that 'we' to go away

Reply:
So: "Rewritten using the abstract syntax described in section \ref{sec:semantics}, get the following big step semantic:"